views/index.jade

```diff
@@ -1,13 +1,14 @@
 extends layout
 
 block content
-
+    if user
```

Comment:
This syntax does not do anything. It looks like you used the correct syntax below - 'if (currentUser)'.